assemblies/features/spring-legacy/src/main/feature/feature.xml

```diff
@@ -39,12 +39,12 @@
         </conditional>
     </feature>
 
-    <feature name="spring-dm-web" description="Spring DM Web support" version="${spring.osgi.version}">
```

Comment:
Why removing all these features?
Is `spring-legacy` repository feature really useful?

Reply:
spring-legacy is not included in the patch. However, when building a new version, it should be considered either to remove spring-legacy entirely or at least to comment out unsecure parts.

Reply:
The spring-legacy repository is not included in the patch. For new builds, however, it should be considered to remove spring-legacy entirely or at least remove unsecure features.

Reply:
I have replaced the previously removed unsecure features by empty features so that uninstallation would succeed.